Dockerfile

```diff
@@ -1,12 +1,14 @@
 FROM ubuntu:18.04
 
+COPY . style-analyzer
+
 RUN apt-get update && \
     apt-get install -y --no-install-suggests --no-install-recommends ca-certificates locales \
-      libxml2 libxml2-dev libsnappy1v5 libsnappy-dev gcc g++ wget \
+      libgomp1 libxml2 libxml2-dev libsnappy1v5 libsnappy-dev gcc g++ wget \
       python3 python3-dev python3-distutils && \
     wget -O - https://bootstrap.pypa.io/get-pip.py | python3 && \
-    pip3 install --no-cache-dir sourced-ml && \
-    pip3 uninstall -y pyspark && \
```

Comment:
`pyspark` is required to pass tests inside Docker:
```
======================================================================
ERROR: lookout.style.typos.tests.test_analyzer (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: lookout.style.typos.tests.test_analyzer
Traceback (most recent call last):
  File "/usr/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/usr/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/style-analyzer/lookout/style/typos/tests/test_analyzer.py", line 11, in <module>
    from lookout.style.typos.analyzer import IdTyposAnalyzer
  File "/style-analyzer/lookout/style/typos/analyzer.py", line 6, in <module>
    from sourced.ml.algorithms import uast2sequence
  File "/usr/local/lib/python3.6/dist-packages/sourced/ml/algorithms/__init__.py", line 4, in <module>
    from sourced.ml.algorithms.uast_ids_to_bag import UastIds2Bag, uast2sequence
  File "/usr/local/lib/python3.6/dist-packages/sourced/ml/algorithms/uast_ids_to_bag.py", line 7, in <module>
    from sourced.ml.utils import IDENTIFIER
  File "/usr/local/lib/python3.6/dist-packages/sourced/ml/utils/__init__.py", line 5, in <module>
    from sourced.ml.utils.spark import add_spark_args, create_spark, get_spark_memory_config, \
  File "/usr/local/lib/python3.6/dist-packages/sourced/ml/utils/spark.py", line 9, in <module>
    import pyspark  # noqa
ModuleNotFoundError: No module named 'pyspark'


======================================================================
ERROR: lookout.style.typos.tests.test_analyzer_model (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: lookout.style.typos.tests.test_analyzer_model
Traceback (most recent call last):
  File "/usr/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/usr/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/style-analyzer/lookout/style/typos/tests/test_analyzer_model.py", line 5, in <module>
    from lookout.style.typos.model import IdTyposModel, NODE_ID_COLUMN
  File "/style-analyzer/lookout/style/typos/model.py", line 8, in <module>
    from sourced.ml.algorithms import TokenParser
  File "/usr/local/lib/python3.6/dist-packages/sourced/ml/algorithms/__init__.py", line 4, in <module>
    from sourced.ml.algorithms.uast_ids_to_bag import UastIds2Bag, uast2sequence
  File "/usr/local/lib/python3.6/dist-packages/sourced/ml/algorithms/uast_ids_to_bag.py", line 7, in <module>
    from sourced.ml.utils import IDENTIFIER
  File "/usr/local/lib/python3.6/dist-packages/sourced/ml/utils/__init__.py", line 5, in <module>
    from sourced.ml.utils.spark import add_spark_args, create_spark, get_spark_memory_config, \
  File "/usr/local/lib/python3.6/dist-packages/sourced/ml/utils/spark.py", line 9, in <module>
    import pyspark  # noqa
ModuleNotFoundError: No module named 'pyspark'


======================================================================
```

Reply:
This must be fixed in sourced-ml first, we must not import any Spark from algorithms

Reply:
Refactoring is needed I guess